fossunited/tests/utils.py

```diff
@@ -0,0 +1,341 @@
+from datetime import datetime, timedelta
+
+import frappe
+from faker import Faker
+
+from fossunited.doctype_ids import (
+    CHAPTER,
+    CITY_COMMUNITY,
+    EVENT,
+    EVENT_RSVP,
+    RSVP_RESPONSE,
+    USER_PROFILE,
+)
+
+fake = Faker()
+
+
+def generate_test_chapter(**kwargs):
```

Comment:
nitpicky naming comment - Can we rename these to `insert_` instead of `generate_`? `generate_` or `create_` usually mean that something will be returned that the caller has to handle. But these functions insert the relevant object into the database before returning them.

Reply:
in my opinion, generate feels better 😅 the way i name methods is largely influenced by this document: https://gist.github.com/18alantom/d9f0565c0f42d6a71311d4a3093a1331#functions

If insert is the better way to do naming in this instance, by all means we should go ahead and do it